vscode-sbcl-clpm.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Getting started with SBCL and CLPM using VSCode
+---
+
+This page is meant to provide an introduction to using
+[VSCode](https://code.visualstudio.com/) as a Lisp IDE with
+[SBCL](http://sbcl.org) as the Common Lisp implementation and
+[CLPM](https://clpm.dev) as the package manager.  Please feel free to
+reach me at hello at smashedtoatoms dot com if you run into trouble.
```

Comment:
That's real nice but what about: we invite them to open an issue here (given you watch them and/or we'll ping you) or on the upstream projects, and they can reach the community on Discord and Reddit (and we could maybe give your handle(s) as a footnote).

Reply:
That's a great suggestion.